lte/gateway/deploy/roles/bazel/tasks/main.yml

```diff
@@ -17,17 +17,21 @@
     state: link
     force: yes
     follow: false
+  when: full_provision
```

Comment:
We don't want to do this in preburn because the links are broken then I guess?

Reply:
This is more a bug than a feature, tbh. While I don't know how these links behave in the preburn/full_provision interplay, I initially thought that some other machine did use the Bazel role as well. But that is not the case, so I reverted the changes. It doesn't really count to the build time and could be moved to preburn with the next image.